### ISSUE TITLE
Playlist Clear feature

### ISSUE THREAD

**Issue by ClareCat** [klnusbaum/UDJ-Desktop-Client#63]
Make it so you can clear the playlist completely.

**Comment by klnusbaum** [comment]
You actually are able to do this, it's just remove is completely broken at the moment. Normally, you just multi select everything and then do a remove. But I've broken remove. I'll fix this. Do you think it would be good to even just add a button that says "clear playlist"?

**Comment by ClareCat** [comment]
Yup.  Imagine 1000+ songs get queued... I don't want to have to scroll to select them all =)
